drivers/nrf24l01/nrf24l01test.py

```diff
@@ -1,14 +1,40 @@
-"""Test for nrf24l01 module."""
-
-import struct
+"""Test for nrf24l01 module. Portable between MicroPython targets."""
+try:
+    import struct
+except ImportError:
+    import ustruct as struct
 import utime
 from machine import Pin, SPI
 from nrf24l01 import NRF24L01
+from sys import platform
```

Comment:
This looks a bit unconventional, I would have used `import sys` and `sys.platform`.